pallets/runtime/tests/src/confidential_test.rs

```diff
@@ -73,3 +84,59 @@ fn range_proof_we() {
         true
     );
 }
+
+#[test]
+fn account_create_tx() {
+    ExtBuilder::default()
+        .build()
+        .execute_with(account_create_tx_we);
+}
+
+fn account_create_tx_we() {
+    let alice = AccountKeyring::Alice.public();
+    let alice_id = register_keyring_account(AccountKeyring::Alice).unwrap();
+
+    let valid_asset_ids: Vec<AssetId> = vec![1, 2, 3]
+        .iter()
+        .map(|id| AssetId::from(id.clone()))
+        .collect();
+    Confidential::store_valid_asset_ids(Origin::signed(alice), valid_asset_ids.clone());
```

Comment:
These asset ids should NOT be updated by Alice.

Reply:
I am assuming this is only for the test

Reply:
Yep, this was kinda a negative test ;)